src/components/table.ts

```diff
@@ -9,7 +9,7 @@ export module Components {
     wantsHeightArr: boolean[];
   }
 
-  export type _IterateLayoutResult = {
+  type _IterateLayoutResult = {
```

Comment:
If it's only being used in one place, it's probably way not worth it to abstract it out to a type